lua/cscope/stack_view/hl.lua

```diff
@@ -87,6 +102,16 @@ M.refresh = function(buf, root)
 			api.nvim_buf_add_highlight(buf, ns, "Number", buf_lnum, lnum_s, lnum_e)
 			api.nvim_buf_add_highlight(buf, ns, "Delimiter", buf_lnum, bc_s, bc_e)
 		else
+			-- all the rest are comments
+			api.nvim_buf_add_highlight(buf, ns, "Comment", buf_lnum, 0, -1)
+		end
+		buf_lnum = buf_lnum - 1
+	end
+
+	-- all the lines below the cursor are comments for sure
+	buf_lnum = cursor_lnum
+	while buf_lnum < buf_lnum_end do
+		if buf_lnum > 0 then
```

Comment:
can this `if` check avoided if `buf_lnum` is initialized with `cursor_lnum+1` at ln 112?